client/project_policy.go

```diff
@@ -1,7 +1,7 @@
 package client
 
 type Policy struct {
-	Id                          string  `json:"id"`
+	Id                          string  `json:"id" tfschema:"-"`
```

Comment:
The "Id" isn't used for resource_project_policy - instead the project id is used.
Ignoring this field.